contrib/systemd/system/zfs-mount.service.in

```diff
@@ -12,4 +12,5 @@ Before=local-fs.target
 [Service]
 Type=oneshot
 RemainAfterExit=yes
-ExecStart=@sbindir@/zfs mount -a
+ExecStart=@sbindir@/zfs mount -a ${VERBOSE_MOUNT:+-v} ${DO_OVERLAY_MOUNTS:+-O} ${MOUNT_EXTRA_OPTIONS}
```

Comment:
My only concern with this is that `VERBOSE_MOUNT="no"` and `DO_OVERLAY_MOUNT="no"`.  That means both these options will be enabled when they should be disabled.  How about limiting this to just `MOUNT_EXTRA_OPTIONS` which will work correctly.